ndm-exporter/collector/seachest.go

```diff
@@ -185,8 +211,30 @@ func (sc *SeachestMetricData) getSeachestData() error {
 		klog.Errorf("error fetching basic disk info using seachest. %s", seachest.SeachestErrors(err))
 		return fmt.Errorf("error getting seachest data for metrics. %s", seachest.SeachestErrors(err))
 	}
+
 	sc.TempInfo.TemperatureDataValid = sc.SeachestIdentifier.GetTemperatureDataValidStatus(driveInfo)
 	sc.TempInfo.CurrentTemperature = sc.SeachestIdentifier.GetCurrentTemperature(driveInfo)
+	sc.TempInfo.LowestTemperature = sc.SeachestIdentifier.GetLowestTemperature(driveInfo)
+	sc.TempInfo.HighestTemperature = sc.SeachestIdentifier.GetHighestTemperature(driveInfo)
+	sc.Capacity = sc.SeachestIdentifier.GetCapacity(driveInfo)
+	sc.RotationRate = sc.SeachestIdentifier.GetRotationRate(driveInfo)
+	sc.RotationalLatency = sc.SeachestIdentifier.GetRotationalLatency(driveInfo)
+	sc.TotalBytesRead = sc.SeachestIdentifier.GetTotalBytesRead(driveInfo)
+	sc.TotalBytesWritten = sc.SeachestIdentifier.GetTotalBytesWritten(driveInfo)
+	sc.DeviceUtilization = sc.SeachestIdentifier.GetDeviceUtilizationRate(driveInfo)
+	sc.PercentEnduranceUsed = sc.SeachestIdentifier.GetPercentEnduranceUsed(driveInfo)
+
+	klog.V(4).Infof("Device is : %v", sc.SeachestIdentifier.DevPath)
+	klog.V(4).Infof("Rotation rate is %v", sc.RotationRate)
+	klog.V(4).Infof("Rotational latency is %v", sc.RotationalLatency)
+	klog.V(4).Infof("Current temperature is %v", sc.TempInfo.CurrentTemperature)
+	klog.V(4).Infof("Lowest temperature is %v", sc.TempInfo.LowestTemperature)
+	klog.V(4).Infof("Highest temperature is %v", sc.TempInfo.HighestTemperature)
+	klog.V(4).Infof("Capacity is %v", sc.Capacity)
+	klog.V(4).Infof("Total bytes read is %v", sc.TotalBytesRead)
+	klog.V(4).Infof("Total bytes written is %v", sc.TotalBytesWritten)
+	klog.V(4).Infof("Device utilization rate is %v", sc.DeviceUtilization)
+	klog.V(4).Infof("Endurance used is %v", sc.PercentEnduranceUsed)
```

Comment:
nit: you can use identifier +v to print fields of sc. If its pointer you need to implement String() method

Reply:
If we do that, we won't be able to give a description of the field during logging, right?